projects/1.16.1/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2083 @@
+    "tetra.stats.speed": "速度",
+    "tetra.stats.speed.tooltip": "影响物品可以被挥动得多快\n \n§e%s §7秒的挥动间隔\n§e%s§7 倍的采集效率",
+    "tetra.stats.speed.tooltip_extended": "挥动“冷却”由屏幕中央的指示条显示。可以在冷却完毕前挥动，但是攻击会造成更少的伤害。横扫或急速打击之类的效果只有在攻击指示条完全充满时才会被触发。§7速度值相当于采集效率的倍率。取决于大于或小于 1,它会增加或减少方块被破坏的速度。倍率计算方式如下：§f§o倍率 = 速度 * 0.5 + 0.5§r§7",
+    "tetra.stats.draw_speed": "拉弓耗时",
+    "tetra.stats.draw_speed.tooltip": "这个物品需要 §e%s §r秒来拉满",
+    "tetra.stats.draw_speed_normalized": "拉弓耗时",
+    "tetra.stats.draw_speed_normalized.tooltip": "将拉弓速度改变 §e%s 秒",
+    "tetra.stats.ability_speed": "冷却",
+    "tetra.stats.ability_speed.tooltip": "这个物品的特殊功能有 §e%s §r秒的基础冷却",
+    "tetra.stats.ability_speed_normalized": "冷却",
+    "tetra.stats.ability_speed_normalized.tooltip": "将基础冷却改变 §e%s §r秒",
+    "tetra.stats.speed_normalized": "速度",
+    "tetra.stats.speed_normalized.tooltip": "将攻击速度改变 §e%s",
+    "tetra.stats.attack_damage": "伤害",
+    "tetra.stats.attack_damage.tooltip": "造成 §e%s §r伤害",
+    "tetra.stats.attack_damage_normalized": "伤害",
+    "tetra.stats.attack_damage_normalized.tooltip": "将攻击伤害改变 §e%s 点",
+    "tetra.stats.draw_strength": "力量",
+    "tetra.stats.draw_strength.tooltip": "完全拉开时射出的弹射物会造成 §e%s - %s§r 的伤害，并具有每刻 §e%s§r 个方块的初速度",
+    "tetra.stats.draw_strength.tooltip_extended": "伤害计算基于原版的弹射物行为。伤害的下限取决于模块与升级，上限是：§f§o上限 = 下限 * 1.5 + 1§r§7。由不完全拉开的弓射出的弹射物伤害基于下限计算，例如：从 50%% 拉开的弓射出的弹射物会造成 50%% 的伤害下限。",
+    "tetra.stats.ability_damage": "伤害",
+    "tetra.stats.ability_damage.tooltip": "这个物品的特殊功能有 §e%s 点的基础伤害",
+    "tetra.stats.bleeding": "流血",
+    "tetra.stats.bleeding.tooltip": "攻击生物时有 30%% 的概率使它们以 §e%s§r 的伤害持续流血 2 秒",
+    "tetra.stats.backstab": "背刺",
+    "tetra.stats.backstab.tooltip": "从背后击中目标会造成致命一击，造成 §e%s§r 点额外伤害",
+    "tetra.stats.armor": "护甲",
+    "tetra.stats.armor.tooltip": "持有时提供 §e%s§r 点护甲值",
+    "tetra.stats.toughness": "坚韧",
+    "tetra.stats.toughness.tooltip": "在持有时提供 §e%s§r 点坚韧值",
+    "tetra.stats.blocking": "防御",
+    "tetra.stats.blocking.tooltip": "按住右键可以防御受到的攻击和弹射物",
+    "tetra.stats.blocking_duration.tooltip": "按住右键可以防御受到的攻击和弹射物，最多持续 §e%s§r 秒",
+    "tetra.stats.blocking.tooltip_extended": "如果这个物品有第二个特殊功能，那么那个功能可以通过按下 shift + 右键，或在右键松开时按住 shift（这会触发一个延长了 50%% 的冷却）。",
+    "tetra.stats.blocking_reflect": "反伤",
+    "tetra.stats.blocking_reflect.tooltip": "§e§7防御近战伤害时有 §e%s§r 的概率对攻击者造成 §e%s§r 的伤害，并产生击退效果。",
+    "tetra.stats.bashing": "猛击",
+    "tetra.stats.bashing.tooltip": "按右键可以猛击生物。猛击会击退生物，击晕它并造成 §e100%%§r 的伤害\n \n§7击退倍率：§e%s§r\n§7击晕持续时间：§e%s§7 秒",
+    "tetra.stats.throwable": "可投掷",
+    "tetra.stats.throwable.tooltip": "可以按住右键来投掷。击中实体时造成 §e%s%%§r 的伤害。击中方块时，如果物品的采集等级足够高并且挖掘速度大于方块硬度，击中方块时会破坏此方块。",
+    "tetra.stats.jab": "猛刺",
+    "tetra.stats.jab.tooltip": "副手持有时可以右键攻击生物，造成 §e%s§r 点伤害并产生击中效果。触发等于物品攻击力的冷却时间。",
+    "tetra.stats.armorPenetration": "穿甲",
+    "tetra.stats.armorPenetration.tooltip": "击中目标将造成至少 §e%s§r 点伤害，无视护甲",
+    "tetra.stats.unarmoredDamage": "切割",
+    "tetra.stats.unarmoredDamage.tooltip": "攻击无护甲的生物时造成 §e%s§r 的额外伤害",
```

|Feedback Priority|
|---|
|🌫 Dust / Non-blocking, “take it or leave it”|

```suggestion
    "tetra.stats.unarmoredDamage.tooltip": "攻击无护甲的生物时造成 §e%s§r 点的额外伤害",
```

本段各行描述需要统一行尾句号的去留。如果有我没看出来的「点」相关问题，也最好改一改。